src/project/internal/projectaudiosettings.h

```diff
@@ -25,18 +25,23 @@
 #include <memory>
 #include <string>
 
+#include <QJsonArray>
+
 #include "modularity/ioc.h"
 #include "playback/iplaybackconfiguration.h"
+#include "context/iglobalcontext.h"
 #include "types/ret.h"
 #include "engraving/infrastructure/mscreader.h"
 #include "engraving/infrastructure/mscwriter.h"
+#include "async/asyncable.h"
 
 #include "../iprojectaudiosettings.h"
 
 namespace mu::project {
-class ProjectAudioSettings : public IProjectAudioSettings
+class ProjectAudioSettings : public IProjectAudioSettings, public async::Asyncable
 {
     INJECT_STATIC(playback::IPlaybackConfiguration, playbackConfig)
+    INJECT_STATIC(context::IGlobalContext, globalContext)
```

Comment:
This is wrong. ProjectAudioSettings must not know anything about GlobalContext. If we want to have audio settings relative to parts/notations, we should re-design this class (probably make it a member of the Notation facade). But this is a pretty big task and should be addressed separately (and not within the scope of this PR)